packages/git-extractor/src/routes/github/api.ts

```diff
@@ -50,6 +50,11 @@ function requestAxios(
     const snakeCaseRequestName = requestName.toLowerCase().replace(/\s/g, "_");
     meter.incrementCounter(`github_request_${snakeCaseRequestName}`, 1);
 
+    // To keep track of how many binary files we are actually trying to request SHAs for
```

Comment:
Adding metrics here because I'm not sure if we retry the "get commits of file" request sometimes and that artifically inflates the numbers we see in logging, so I want to know the actual number of files we are requesting across all sandboxes.